Tests/Api/OptimizationTests.cs

```diff
@@ -119,8 +119,9 @@ public void EstimateOptimization()
             Assert.GreaterOrEqual(estimate.Balance, 0);
         }
 
-        [Test]
-        public void CreateOptimization()
+        [TestCase(1)]
```

Comment:
This test case is failing, the returned `optimization` doesn't have the targetValue set